src/App/App.js

```diff
@@ -13,8 +13,11 @@ class App extends Component {
   render() {
     return (
       <main className='App'>
-        <header>{this.state.user.id ? 'Log Out' : 'Log In'}</header>
-        <h1>Rancid Tomatillos</h1>
+        <header className='header'>
+          <h1>Rancid Tomatillos</h1>
+          {this.state.user.id ? 'Log Out' : 'Log In'}
+          <section className='greeting'>{'Welcome, ' + this.state.user.name + '!'}</section>
```

Comment:
Does this not need a condition? What if there is no user.name property in our state?